larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -470,32 +537,180 @@ inline void GetConvParamsType(const TfLiteBConv2DParams& conv_params,
   op_params.quantized_activation_max = conv_params.output_activation_max;
 }
 
-template <typename SrcScalar, typename TBitpacked, typename DstScalar>
-void EvalOpt(TfLiteContext* context, TfLiteNode* node,
-             TfLiteBConv2DParams* params) {
-  const auto* input = GetInput(context, node, 0);
-  const auto* filter = GetInput(context, node, 1);
+using compute_engine::core::OutputTransform;
+using compute_engine::core::OutputTransformBase;
+
+// Fill the parts of the OutputTransform struct that are common to each
+// destination type
+void GetBaseParams(TfLiteContext* context, TfLiteNode* node,
```

Comment:
I think all of the helper functions added here, need to move to a seprate file (`quantization_utils.h`?). the bconv2d file is already huge.

Reply:
I agree that it makes sense to move this. However these functions need to access our Bconv2dparams struct. Shall we move that struct to a separate header file as well?

Reply:
yes, I think so

Reply:
I moved these functions and the struct to their own separate files in 654ef34